css/amp.css

```diff
@@ -167,6 +167,16 @@ html.i-amphtml-singledoc.i-amphtml-ios-embed-sd > body {
   display: inline-block;
 }
 
+.i-amphtml-fade-out .i-amphtml-blur {
```

Comment:
`i-amphtml-layout` class is added after `layoutCallback()`, so can we avoid a custom CSS class altogether and do instead:

```css
.i-amphtml-layout > .i-amphtml-blur {
  opacity: 0 !important;
}

.i-amphtml-blur {
  transition: ...;
  filter: ...;
  transform: ...;
}
```